backend/src/main.ts

```diff
@@ -4,7 +4,8 @@ import {AppModule} from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.enableCors({
-    origin: [process.env.FRONTEND_URL],
+    origin: true,
+    methods: 'GET,HEAD,PUT,PATCH,POST,DELETE,OPTIONS',
```

Comment:
remove the changes made in this file.

Reply:
Yep , sorry for not telling you that day to change it , @ANIRUDHNARANG0  , revert to how  itused to be...